deployment/secondary_architectures/s390.md

```diff
@@ -0,0 +1,70 @@
+---
+title: s390(x) architectures
+subsection: secondary_architectures
+
+---
+
+# s390(x)
+
+s390 is 31-bit-address/32-bit-data and s390x is 64 bit computing architecture. s390(x) is [big endian](https://en.wikipedia.org/wiki/Endianness) and used in IBM mainframe computers which is also known as [IBM System z](https://en.wikipedia.org/wiki/IBM_System_z).
+
+## Building your application for s390(x) architecture
+
+
+To build a package for s390(x) architecture in Fedora, use s390-koji command:
+
+````
+$ s390-koji --scratch build rawhide srpm_pkg1
+````
+
+The above command will start a scratch build of package srpm_pkg1 against Fedora rawhide for s390 and s390x architectures. Use "s390-koji \-\-help" in console to see all available options.
+
+To build against other Fedora branches (e.g. F23, F24), specify specific branch name to s390-koji in argument:
+
+```
+s390-koji --scratch build F24 srpm_pkg1
+````
+
+To package application exclusively for s390(x) archiecture, use ExclusiveArch tag in [RPM spec file](https://fedoraproject.org/wiki/How_to_create_an_RPM_package#Creating_a_SPEC_file)
+
+````
+ExclusiveArch:  s390, s390x
+````
+
+## Example
+
+In Secondary Architectures section, we saw how to get/create iprutils srpm as an example. Now, you already have srpm (src.rpm) file of iprutils.
+The following example will demonstrate how to build an application for s390(x) architecture from srpm:
+
+````
+# Initiate iprutils package build for s390(x) architecture
+$ s390-koji build --scratch rawhide iprutils-2.4.11.1-1.fc25.src.rpm 
+  Created task: 2183374
+  Task info: http://s390.koji.fedoraproject.org/koji/taskinfo?taskID=2183374
+  ....
+
+````
+For each build, a Task ID is created in s390 koji. You can view build status and it's log by going to the url provided in Task info.
+
+
+## Viewing koji builds
+
+Builds specific to s390(x) are hosted at [http://s390.koji.fedoraproject.org/](http://s390.koji.fedoraproject.org/). UI is very similar to [primary koji](http://koji.fedoraproject.org/).
+ 
+## Getting access to s390(x) machine
+
+It is possible that to fix/test some issues/features s390(x) quickly, you require access to a s390 machine. Please send an email either directly to [Dan Horák](https://fedoraproject.org/wiki/User:Sharkcz) at <dan@danny.cz> or ask on [s390(x) Mailing List](https://lists.fedoraproject.org/admin/lists/s390x.lists.fedoraproject.org/)
```

Comment:
I would prefer to modify it as above. User a reference from Secondary Arches wiki. Not directly.